text_to_x/text_to_embeddings.py

```diff
@@ -0,0 +1,306 @@
+from collections import Counter
+import numpy as np
+import pandas as pd
+from scipy import sparse
+from scipy.sparse import linalg 
+import umap
+    
+    
+class SvdEmbeddings():
+    '''
+    train and operate with PMI-SVD embeddings
+    
+    Examples:
+    - training with a manually set class instance
+        your_instance = SvdEmbeddings(texts)
+        your_instance.train(front_window = int, back_window = int, embedding_dim = int)
+    
+    - training with default settings:
+        your_instance = svd2vec_run_default(texts)
+        
+    - visualizing the fit
+    '''
+    
+    def __init__(self, texts, texts_colname = 'lemma'):
+        '''
+        texts (iterable | TextToToken): input list of dataframes acquired from text_to_x.text_to_token. The "lemma" column is used. 
+        texts_colname (string): column from TextToToken DataFrame to process ("lemma" / "stem" / "word")
+        '''
+
+        self.model = {}
+        self.tok2ind = {}
+        self.ind2tok = {}
+        self.unigram_counts = {}
+        
+        # check if input is TextToToken or simialr
+        if not isinstance(texts[0], pd.DataFrame):
+            raise ValueError("Input texts are not in a DataFrame or a list of DataFrames, please use TextToToken for preprocessing")
+        
+        # unpack lemmas 
+        lemmas_to_list = []
+        for doc in texts:
+            doc_to_list = doc.loc[:, texts_colname].tolist()
+            lemmas_to_list.append(doc_to_list)
+
+        self.texts = lemmas_to_list
+      
+    
+    def train(self, back_window, front_window, embedding_dim):
+        '''
+        train svd embeddings from texts
+        
+        - back_window (int): max number of words to look behind in a skipgram 
+        - front_window (int):, max number of words to look forward in a skipgram 
+        - embedding_dim (int): number of dimensions to fit the embeddings to
+        '''
+        
+        ### UNIGRAMS
+        tok2indx = dict()
+        unigram_counts = Counter()
+        for ii, text in enumerate(self.texts):
+            for token in text:
+                unigram_counts[token] += 1
+                if token not in tok2indx:
+                    tok2indx[token] = len(tok2indx)
+        indx2tok = {indx:tok for tok,indx in tok2indx.items()}
```

Comment:
```suggestion
        indx2tok = {indx: tok for tok, indx in tok2indx.items()}
```

In general, I have changed the package to be flake8 compliant (you can install flake8 as a linter. You are free to use it as a style guide. Otherwise, I can fix the script as well.